### PR TITLE
add tests for plot_roc

### LINKED ISSUE #29: write function and test for plot_roc

### PR DESCRIPTION
**Pull request by taohuang-ubc**
i now only manage to get one unit test for plot_roc. because I cant find ways to get the attributes out of matplotlib because it is not even an object. I PR for now to meet the deadline. I will try to improve later.

### CODE REVIEW

File: pymlviz/plot_roc.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        
+    if y_valid.shape[0] != X_valid.shape[0]:
+        raise Exception("Sorry, X_valid and y_valid should have the same number of rows.")
+
+    probs = model.predict_proba(X_valid)
+    preds = probs[:,1]
+    fpr, tpr, threshold = metrics.roc_curve(y_valid, preds)
+    roc_auc = metrics.auc(fpr, tpr)
+    
+    plt.title('ROC curve')
```
Comment (flizhou) [inline pymlviz/plot_roc.py]: Increase the title font to better visualize the plot
```suggestion
    plt.title('ROC curve', fontsize=18)
```

File: pymlviz/plot_roc.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    plt.plot(fpr, tpr, 'b', label = 'AUC = %0.2f' % roc_auc)
+    plt.legend(loc = 'lower right')
```
Comment (flizhou) [inline pymlviz/plot_roc.py]: Increase the legend font to better visualize the plot
```suggestion
    plt.legend(loc = 'lower right', fontsize=12)
```

File: pymlviz/plot_roc.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    plt.plot([0, 1], [0, 1],'r--')
+    plt.xlim([0, 1])
+    plt.ylim([0, 1])
+    plt.ylabel('True Positive Rate')
+    plt.xlabel('False Positive Rate')
```
Comment (flizhou) [inline pymlviz/plot_roc.py]: Increase the label font to better visualize the plot
```suggestion
    plt.ylabel('True Positive Rate', fontsize=15)
    plt.xlabel('False Positive Rate', fontsize=15)
```

**Review by flizhou — COMMENTED** [review]
Please review the few changes to the plot

**Comment by taohuang-ubc** [comment]
Fanli, thank you for you suggestions, I think they are very good, and I commit all your suggestions.

**Review by flizhou — APPROVED** [review]
Looks good!

**Pull request merged**